src/Symfony/Component/VarDumper/Dumper/ContextProvider/SourceContextProvider.php

```diff
@@ -52,8 +52,8 @@ public function getContext(): ?array
                 && 'dump' === $trace[$i]['function']
                 && VarDumper::class === $trace[$i]['class']
             ) {
-                $file = $trace[$i]['file'];
-                $line = $trace[$i]['line'];
+                $file = $trace[$i]['file'] ?? null;
+                $line = $trace[$i]['line'] ?? null;
```

Comment:
both lines should fallback to the previous value of the variable, eg `$line = $trace[$i]['line'] ?? $line;`

Reply:
You're right, I've edited to include this.